projects/valtimo/form/src/lib/form.service.ts

```diff
@@ -41,7 +42,9 @@ export class FormService {
     documentId: string
   ): Observable<FormioForm> {
     return this.http.get<FormioForm>(
-      `${this.valtimoApiConfig.endpointUri}form-association/form-definition/${formDefinitionName}?documentId=${documentId}`
+      `${this.valtimoApiConfig.endpointUri}form-association/form-definition/${formDefinitionName}?documentId=${documentId}`, {
+        headers: InterceptorSkipHeader
```

Comment:
Did you test if this still works if there is a form? I'm afraid it might skip adding the token to the request as well.

Reply:
Yes, I tested it and it works for both with form and without form